src/components/datepicker/datePicker-theme.scss

```diff
@@ -8,10 +8,10 @@
   }
 
   .md-datepicker-input-container {
-    border-bottom-color: '{{background-300}}';
+    border-bottom-color: '{{foreground-4}}';
```

Comment:
👍  This matches https://github.com/angular/material/blob/master/src/components/input/input-theme.scss#L5